doc/classes/@GlobalScope.xml

```diff
@@ -783,7 +783,6 @@
 			<return type="RID" />
 			<argument index="0" name="base" type="int" />
 			<description>
-				Create a RID from an int64. This is used mainly from native extensions to build servers.
```

Comment:
This change still needs to be reverted

Reply:
Fixed. I also found a few others that needed to be fixed